community/server/pom.xml

```diff
@@ -159,7 +159,7 @@
 
     <dependency>
       <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-classic</artifactId>
+      <artifactId>logback-core</artifactId>
```

Comment:
@cleishm Netty does not use logback, it uses Slf4j. Debug logging to Slf4j is not a bug, the problem is us having an Slf4j handler on the classpath (eg. logback-classic) without having configured it. In fact, Netty sends the exact same log message to debug with the fix in this PR, the difference being that j.u.l is piped to be handled by us.

@tinwelint I've still not understood the reasoning for us building our own logging system - I understand us having our own logging *interfaces*, but I don't understand why we are building our own logging backend. Eg. I wrote that Netty4LoggerFactory that CL mentioned, and it's super frustrating, because we are going to have to do that to every single third party library we pull in, and we are going to keep forgetting and keep having to figure out how each new library does custom logging. 

If we used Logback and configured it properly, it registers itself as a standard handler for Slf4j, and logging would "just work" for any normal java lib we pull in. Alternatively we could build our own Slf4j integration - but again *why* are we doing this when there are really performant alternatives out there already, which we have on the classpath anyway?

Replacing the HTTP logging, which is on the hot path, would mean we also need to match Logbacks performance - keeping in mind that it uses disruptor under the hood to offload logging work. Eg; that's a lot of engineering to do.

Is there an upside to building this in-house that I'm not seeing?
</rant>